data/network/src/androidMain/kotlin/social/androiddev/common/network/UrlStream.kt

```diff
@@ -6,6 +6,6 @@ import io.ktor.client.engine.cio.CIO
 import io.ktor.client.request.get
 import java.io.ByteArrayInputStream
 
-actual suspend fun urlStream(url: String): ByteArrayInputStream = HttpClient(CIO).use {
-    ByteArrayInputStream(it.get(url).body())
+suspend fun String.toUrlStream(): ByteArrayInputStream = HttpClient(CIO).use {
+    ByteArrayInputStream(it.get(this).body())
```

Comment:
I think alternatively a better approach here would be to either:
A) Update the `urlStream` contract so it's available/implemented by android, iOS, and Desktop JVM
B) Remove the contract and create some platform-agnostic solution in `commonMain`

Reply:
This s what I'm trying to say. See #62 how I have done it.